src/v2/Apps/Artists/artistsRoutes.tsx

```diff
@@ -69,6 +71,19 @@ export const artistsRoutes: RouteConfig[] = [
             }
           }
         `,
+        render: ({ Component, props, match }) => {
+          if (!(Component && props)) {
+            // Returning `null` will show the spinner; but `undefined` uses the loading bar
+            return undefined
+          }
+
+          // If the `letter` param isn't part of the nav, 404
+          if (!LETTERS.includes(match.params.letter?.toUpperCase())) {
```

Comment:
I guess this is when going to `artists-starting-with-X` where `X` isn't a single character from a-z? I would have thought the router would match that, I think you can do: `artists-starting-with-:letter([a-zA-Z])` and get that same behavior without this.

Reply:
Oh nice. Yeah I'll do that.